src/Components/NavBar/Menus/MoreNavMenu.tsx

```diff
@@ -33,6 +33,7 @@ export const MoreNavMenu: React.FC<{ width?: number }> = ({ width = 160 }) => {
       <MenuItem href="/artists">Artists</MenuItem>
       <MenuItem href="/shows">Shows</MenuItem>
       <MenuItem href="/institutions">Museums</MenuItem>
+      <MenuItem href="/consign">Consign</MenuItem>
```

Comment:
Should this be added on mobile as well?

Reply:
Not as part of this ticket at least.

(Not all of the desktop CTAs  — join as a gallery, consign your works — are in the mobile nav. But I'll ping folks to be sure. Can follow up in another PR if necessary.)